cookie-banner-rules-list.json

```diff
@@ -1404,16 +1287,12 @@
       "domains": ["lemonde.fr"]
     },
     {
-      "click": { "optIn": "button#_cpmt-accept", "presence": "div#_cpmt" },
-      "cookies": {
-        "optIn": [
-          {
-            "name": "euconsent-v2",
-            "value": "CPgpnrRPgpnrRFzABBFRBdCgAP_AAH_AAAqIH7Nf_X__b3_n-_79__t0eY1f9_7_v-0zjhfdt-8N2f_X_L8X_2M7vF36pr4KuR4ku3bBIQdtHOncTUmx6olVrTPsbk2Mr7NKJ7Pkmnsbe2dYGH9_n93T_ZKZ7______7________________________-_____9____________wP2AJMNS-AC7EscGSaNKoUQIQrCQ6AUAFFAMLRNYQMrgp2VwEeoIGACE1ARgRAgxBRiwCAAQCAJCIgJADwQCIAiAQAAgBUgIQAEbAILACwMAgAFANCxAigCECQgyOCo5TAgIkWignsrAEou9jTCEMosAKBR_RUYCJQggWBkJCwcxgAA"
-          }
-        ]
+      "click": {
+        "optIn": "button#privacy-cp-wall-accept",
+        "presence": "div.privacy-cp-wall"
       },
-      "id": "4aa16185-4697-4bc4-b0b5-6afecd64c77f",
```

Comment:
Same question as @Trikolon above, why does this id need to change?

Reply:
It does not need to change, that is a personal error. Only the CSS selectors needed to be changed.